### DISCUSSION TITLE
Ignore error message temporarily?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [lutris/lutris#4984] [Q&A] [answered]
I've only the option to permanently.

![image](https://github.com/lutris/lutris/assets/42837531/b2f2f62c-fc06-4366-9a5c-4e8ad90a8438)

https://github.com/lutris/lutris/discussions/4979#discussioncomment-6723182

**Comment by danieljohnson2** [comment, marked as answer]
This will be significantly different in 0.5.14; that checkbox is going away. You will not be able to start the game with ESYNC enabled, if the Wine version does not support it.

You need to turn off Esync in the Runner configuration for the game, or for the runner overall. In the glorious future of 0.5.14, a warning will appear next to that option in the configuration, in addition to this dialog.

Hopefully this will make it easier to figure out.

Anyway, it's not useful to turn ESYNC on if it is not supported; if your game works, you didn't need it. Just turn it off for the game at least.
